tests/memarray/README.md

```diff
@@ -0,0 +1,18 @@
+Expected result
+===============
+
+This application should runs a number of tests equal to NUMBER_OF_TESTS (Default 12).
+
+At the beginning of the tests is printed the memory being used by the threads.
```

Comment:
wording, consider `At the beginning of the tests memory usage of each thread is printed.`